src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/MediaPicker/MediaPickerDialog/MediaPickerDialog.js

```diff
@@ -66,24 +66,21 @@ export const MediaPickerDialogCmp = ({onCloseDialog, classes, idInput, t, site,
     const [selectedImages, setSelectedImages] = useState([]);
     const {images, error, loading} = useImagesData(selectedPath);
 
-    if (loading) {
-        return (
-            <section>
-                <CircularProgress/>
-            </section>
-        );
-    }
-
     return (
-        <>
+        <>{
+            loading ?
```

Comment:
Minor: we can use Inline If with logical `&&` operator.

Reply:
done, thanks !